Formula/a/apache-polaris.rb

```diff
@@ -0,0 +1,42 @@
+class ApachePolaris < Formula
+  desc "Interoperable, open source catalog for Apache Iceberg"
+  homepage "https://polaris.apache.org/"
+  url "https://github.com/apache/polaris/archive/refs/tags/apache-polaris-1.0.0-incubating.tar.gz"
+  sha256 "4ed1a13aae04c8bf25982bc059e12d3490cb70ddf3b175dc97a227b46a566e10"
+  license "Apache-2.0"
+
+  depends_on "gradle@8" => :build
+  depends_on "openjdk"
+
+  def install
+    system "gradle", "assemble"
+
+    mkdir "build" do
+      system "tar", "xzf", "../runtime/distribution/build/distributions/polaris-bin-#{version}-incubating.tgz", "--strip-components", "1"
+      libexec.install "admin", "bin", "server"
+    end
+
+    java_env = Language::Java.overridable_java_home_env
+    %w[admin server].each do |script|
+      (bin/"polaris-#{script}").write_env_script libexec/"bin"/script, java_env
+    end
+  end
+
+  service do
+    run [opt_bin/"polaris-server"]
+    keep_alive true
+    error_log_path var/"log/polaris.log"
+    log_path var/"log/polaris.log"
+  end
+
+  test do
+    port = free_port
+    ENV["QUARKUS_HTTP_PORT"] = free_port.to_s
+    ENV["QUARKUS_MANAGEMENT_PORT"] = port.to_s
+    spawn bin/"polaris-server"
+    sleep 30
```

Comment:
Would be nice if we could make this less ad-hoc.

Maybe something like
```suggestion
    polaris_log = var/"log/polaris.log"
    touch polaris_log
    old_log_size = polaris_log.size
    spawn bin/"polaris-server"
    sleep 5 until polaris_log.size > old_log_size
```
Where you can adjust `polaris_log.size > old_log_size` as appropriate (e.g. to `polaris_log.size > old_log_size + 10`).

Reply:
Sounds good!